src/lib.rs

```diff
@@ -141,6 +140,7 @@ pub use xcb::{
 };
 
 pub mod prelude;
+pub mod constants;
```

Comment:
Maybe
```rust
mod constants;
pub use constants::*;
```
instead?

Reply:
Out of curiosity whats the difference between those two? Yours looks cleaner ;)

Reply:
Yours requires users to do `cairo::constants::BLABLA`, mine would be `cairo::BLABLA`